validation/json-validator/src/main/java/com/redhat/mercury/util/schemavalidator/api/SchemaValidator.java

```diff
@@ -0,0 +1,4 @@
+package com.redhat.mercury.util.schemavalidator.api;
```

Comment:
Remove this interface. It is empty and brings no value

Reply:
Ops my mistake I wanted to add the validateJson method in it